### DISCUSSION TITLE
Sensor Fail 7 G7

### DISCUSSION THREAD

**Discussion by annele373** [NightscoutFoundation/xDrip#2975] [General]
Hi, 

I haven't got any readings in the last 7 hours or so, Ive sent the logs, just trying to figure out if I set the xDrip up incorrectly or its a Dexcom issue. 

I have been using xDrip for a while but only recently switched to using it instead of the Dexcom g7 app. 

Dexcom G7 is the device

The phone is Samsung S21 Ultra 5G, android version 13

I checked the settings and restarted the phone but the issue persists. 

I would be very grateful for help/advice/how to avoid this in the future. 

![Screenshot_20230731_195402_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/129327758/4d99ca00-8750-4939-9dab-ccc69698e23a)
![Screenshot_20230731_195434_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/129327758/656d0441-c6b2-4df8-9b0c-9342ee4fea6a)

**Comment by Navid200** [comment]
That looks like a sensor error.
Was it inserted properly?
Did this one work at all?

**Reply by annele373** [reply]
Hi, yes, it worked fine for the first 6 or 7 days, then stopped out of nowhere.

**Reply by kameamea** [reply]
Did you try to connect it to the Dexcom app now after the failure? That's what i do in such cases and if the Dexcom app shows sensor failure Dexcom will replace the sensor

**Reply by annele373** [reply]
oh no I didn't but will do in the future! I thought I couldn't have the dexcom app simultaneously on my phone with xDrip thats set up to source from the sensor directly though?

**Reply by kameamea** [reply]
You either need to stop one of the two apps via settings->apps or simply use two different sensor codes, even non existing ones. Then there is no conflict.

**Reply by annele373** [reply]
oh nice, thanks! Ill do this if it happens again, sensor failures have happened before with g7 sensors, not sure what causes it
